vrc-get-gui/app/(main)/projects/manage/package-list-card.tsx

```diff
@@ -406,10 +406,6 @@ function ManagePackagesHeading({
 				"flex flex-wrap flex-shrink-0 flex-grow-0 flex-row gap-2 items-center"
 			}
 		>
-			<p className="cursor-pointer py-1.5 font-bold flex-grow-0 flex-shrink-0">
-				{tc("projects:manage:manage packages")}
-			</p>
-
```

Comment:
![image](https://github.com/user-attachments/assets/b01d72b0-c296-49ac-8069-bd29c74de040)
How about this?

Reply:
I removed it because several of the other tables don't have headings, and having one here would be a bit jarring

Reply:
It looks better.
but I have to consider "manage" is suitable for the page.
I feel manage means for multiple things, but this is targeting single project.
I'm not native so my feeling might be incorrect though.

> I removed it because several of the other tables don't have headings, and having one here would be a bit jarring

I'm adding heading where unclear, and doesn't have where doesn't needed.

Reply:
> but I have to consider "manage" is suitable for the page.

In Chinese and English the word 'Manage' doesn't explicitly specify whether the object is single or multiple, both cases apply,
I'm not sure how it works in Japanese as